### DISCUSSION TITLE
准备工作 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1021] [Announcements]
# 准备工作 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/prepare/

**Comment by zgfh** [comment]
临时离线包： https://qiniu-download-public.daocloud.io/opensource/tmp-/data/dce-5.0/offline-centos7-v0.5.0-amd64-prerequisite.tgz-929f48a50855a24f1da00c91034b864b
